### PR TITLE
Chore: Remove old work queue health and wq status feature flag

### PR DESCRIPTION
**Pull request by collincchoy**
This PR removes remaining references to work queue health which was deprecated in favor of a new, simpler status field on the WorkQueue object.

This PR also removes usage of the workQueueStatus feature flag in prep for full removal

### CODE REVIEW

File: src/services/WorkspaceWorkQueuesApi.ts [hunk 1]
```diff
@@ -89,10 +89,4 @@ export class WorkspaceWorkQueuesApi extends WorkspaceApi {
 
     return mapper.map('FlowRunResponse', data, 'FlowRun')
   }
-
-  public async getWorkQueueStatus(id: string): Promise<WorkQueueStatus> {
-    const { data } = await this.get<WorkQueueStatusResponse>(`/${id}/status`)
-
-    return mapper.map('WorkQueueStatusResponse', data, 'WorkQueueStatus')
-  }
```
Comment (collincchoy) [inline src/services/WorkspaceWorkQueuesApi.ts:97]: this API is still available server-side, but it's no longer being used in the UI so I'm removing it to reduce confusion between the old work queue status and new work queue status.

today, the status comes directly off the work queue entity like `workQueue.status`

File: src/maps/workPoolQueue.ts [hunk 2]
```diff
@@ -31,6 +32,7 @@ export const mapWorkPoolQueueToWorkPoolQueueResponse: MapFunction<WorkPoolQueue,
     is_paused: source.isPaused,
     concurrency_limit: source.concurrencyLimit,
     priority: source.priority,
+    last_polled: this.map('Date', source.lastPolled, 'string'),
     status: source.status.toUpperCase() as Uppercase<WorkPoolQueueStatus>,
   }
 }
```
Comment (znicholasbrown) [inline src/maps/workPoolQueue.ts:38]: This mapper doesn't make sense to me, when are we mapping a client-side work queue back to an api-side object 🤔

File: src/components/WorkPoolQueueStatusBadge.vue [hunk 3]
```diff
@@ -1,70 +1,29 @@
...
+  ))
 </script>
 
 <style>
 .work-pool-queue-status-badge { @apply
   text-xs
   pl-2
 }
-
-.p-tag.work-pool-queue-status-badge--healthy { @apply
```
Comment (znicholasbrown) [inline src/components/WorkPoolQueueStatusBadge.vue:51]: So the visual indicator these styles gave before is coming from the status icon alone now?

**Comment by znicholasbrown** [comment]
ah shoot sorry didn't realize this was in draft, ignore me

File: src/components/WorkPoolQueueStatusBadge.vue (hunk 3 above)
Comment (collincchoy) [reply src/components/WorkPoolQueueStatusBadge.vue:51]: yup!

https://github.com/PrefectHQ/prefect-ui-library/blob/e59fd70c91245fcddcd470feca59b09ce4ff711e/src/components/WorkPoolQueueStatusIcon.vue#L1

File: src/maps/workPoolQueue.ts (hunk 2 above)
Comment (collincchoy) [reply src/maps/workPoolQueue.ts:38]: y'know. i thought that too. 

and a quick search _looks_ like i can just cull this. will double-check slash remove. brb

File: src/components/WorkPoolQueueStatusBadge.vue (hunk 3 above)
Comment (znicholasbrown) [reply src/components/WorkPoolQueueStatusBadge.vue:51]: thank you!

File: src/maps/index.ts [hunk 4]
```diff
@@ -241,10 +239,7 @@ export const maps = {
   WorkPoolCreate: { WorkPoolCreateRequest: mapWorkPoolCreateToWorkPoolCreateRequest },
   WorkPoolEdit: { WorkPoolEditRequest: mapWorkPoolEditToWorkPoolEditRequest },
   WorkPoolFilter: { WorkPoolFilterRequest: mapWorkPoolFilter },
-  WorkPoolQueue: {
-    WorkPoolQueueResponse: mapWorkPoolQueueToWorkPoolQueueResponse,
```
Comment (collincchoy) [inline src/maps/index.ts:245]: this mapper for WorkPoolQueue -> WorkPoolQueueResponse is unused. removing.

File: src/models/WorkQueue.ts [hunk 5]
```diff
@@ -1,53 +1,20 @@
...
-// deployments have new editable field called work_queue_name
-// flow runs do as well
-
-export class WorkQueue implements IWorkQueue {
-  public readonly id: string
-  public created: Date
-  public updated: Date
-  public name: string
+/** @deprecated Prefer `WorkPoolQueue` */
+export class WorkQueue extends WorkPoolQueue implements IWorkQueue {
```
Comment (collincchoy) [inline src/models/WorkQueue.ts:9]: We can _likely_ remove these old models - WorkQueue, WorkQueueCreate, WorkQueueEdit, WorkQueueFilter (and their respective api responses). 

Doing this "shim" deprecation for now - this PR is already a bit large.

File: src/maps/workPoolQueue.ts (hunk 2 above)
Comment (collincchoy) [reply src/maps/workPoolQueue.ts]: killed that. there was on for `WorkQueue` -> `WorkQueueResponse` also 🤨 . killed that one too.

**Review by znicholasbrown — APPROVED** [review]

**Pull request merged**